cdc/sink/codec/maxwell.go

```diff
@@ -257,11 +318,9 @@ func (d *MaxwellEventBatchEncoder) EncodeDDLEvent(e *model.DDLEvent) (*MQMessage
 	var versionByte [8]byte
 	binary.BigEndian.PutUint64(versionByte[:], BatchVersion1)
 	keyBuf.Write(versionByte[:])
```

Comment:
We need to remove versionByte from the DDL output too.

Reply:
done